LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2017 Stanislav Seletskiy & Egor Kovetskiy
+Copyright © 2017-2020 Stanislav Seletskiy & Egor Kovetskiy
```

Comment:
This could be interpreted that copyright has been relinquished from 2020 onwards

Reply:
I don't believe so, as long as it's the current and not past year it remains with the authors. 

I mean it's a total obsessive compulsive nit pick, to be fixing other folks copyrights year ranges, so perhaps I should get it checked out 😃